frontend-cert/react-projects/reusable-footer/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should have a `div` with an `id` of `root`.
+1. You should have a `script` tag that specifies you are using `babel` to compile `JSX`.
```

Comment:
We don't need user stories for these because the basic boilerplate will already been supplied for them